src/assets/scss/app.scss

```diff
@@ -446,8 +453,6 @@ $form-group-padding-bottom: 15px;
 .empty-table-message {
   color: #555;
   font-size: 15px;
-  // Unless the message is quite long, we want it to fit on one line.
-  max-width: 650px;
```

Comment:
The value of `max-width` in the `text-block` mixin will scale with the font size, so this line is no longer needed.